### DISCUSSION TITLE
Error to get user api

### DISCUSSION THREAD

**Discussion by queza85** [exoplatform/exo-community#78] [Q&A]
Hi all.

I would like to ask this error when portlet call member information in the space page.
"org.exoplatform.social.core.storage.cache.model.data.ListIdentitiesData.getIds()"
![image](https://github.com/exoplatform/exo-community/assets/2861131/53918f12-8a4d-4529-b16f-ac26e2d78f65)

when portal call this api, exo server returns from DB directly? or it is related with elastic search?

If it is related with elastic search, how can I access and query to ES in the docker?
Can I get any document or example?

**Comment by rdenarie** [comment]
Hello, Can you ask your question on https://community.exoplatform.org ?
You will have more visibility to have an answer.

Thanks

**Reply by queza85** [reply]
Thanks. but when I try to access here(https://community.exoplatform.org/), I cannot reach this site.

**Reply by rdenarie** [reply]
sorry, it is https://community.exoplatform.com, my bad :)

**Reply by queza85** [reply]
thanks!!
